cmake/modules/OCApplyCommonSettings.cmake

```diff
@@ -0,0 +1,12 @@
+function(apply_common_settings targetNamme)
+    target_compile_definitions(${targetNamme}
+        PRIVATE QT_NO_CAST_TO_ASCII
```

Comment:
4 spaces indentation is the default. Might also be easier if you just repeat `PRIVATE` on every line.